scripts/new_bare_repo

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
```

Comment:
Out of scope for this PR, I'm thinking we might want to add an engineering tools script that checks if kinit/aklog needs to be run (and optionally, runs it). This is doable using the output of the `klist` command.

Would something like this have been useful when writing this script?

Reply:
Personally, I am not even sure when to run kinit or not... It's usually something I only run when I get an error message with git that I can't explain otherwise

Reply:
I've added an issue for this, I might get around to it at some point: https://github.com/pcdshub/engineering_tools/issues/147